tests/models/wav2vec2_conformer/test_modeling_wav2vec2_conformer.py

```diff
@@ -210,6 +226,19 @@ def create_and_check_model_with_adapter_for_ctc(self, config, input_values, atte
             result.logits.shape, (self.batch_size, self.adapter_output_seq_length, self.vocab_size)
         )
 
+    def create_and_check_causal_fbank_model(self, config, input_values, attention_mask):
+        config.add_adapter = False
+        config.skip_feature_encoder = True
+        config.skip_encoder_layer_norm = True
+        config.skip_pos_conv_embed = True
+        config.non_causal_depth_wise_conv = False
```

Comment:
Is it possible to have any combination of `skip_feature_encode`, `skip_encoder_layer_norm`, `skip_pos_conv_embed` and `non_causal_depth_wise_conv` and still have a sucessful forward pass? 

It's not necessary to do all combinations, but if it is possible, one test when some are true and some are false should be added. If not possible, then there should be input validation in the config.

Reply:
Done !